packages/lb-components/src/components/pointCloudView/PointCloud2DView.tsx

```diff
@@ -183,21 +191,25 @@ const PointCloud2DView = ({ currentData, config }: IA2MapStateProps) => {
       style={hiddenData ? { display: 'none' } : { display: 'flex', minHeight: 200, maxHeight: 500 }}
     >
       <div className={getClassName('point-cloud-2d-image')} ref={ref}>
-        <AnnotationView
-          src={mappingData?.url ?? ''}
-          annotations={annotations2d}
-          size={size}
-          ref={viewRef}
-          globalStyle={{
-            display: hiddenData ? 'none' : 'block',
-          }}
-          afterImgOnLoad={afterImgOnLoad}
-          zoomInfo={{
-            min: 0.01,
-            max: 1000,
-            ratio: 0.4,
-          }}
-        />
+        {cuboidBoxIn2DView ? (
+          <AnnotationView
+            src={mappingData?.url ?? ''}
+            annotations={annotations2d}
+            size={size}
+            ref={viewRef}
+            globalStyle={{
+              display: hiddenData ? 'none' : 'block',
+            }}
+            afterImgOnLoad={afterImgOnLoad}
+            zoomInfo={{
+              min: 0.01,
+              max: 1000,
+              ratio: 0.4,
+            }}
+          />
+        ) : (
+          <Rect2DOperationView size={size} mappingData={mappingData} checkMode={checkMode} />
```

Comment:
`Rect2DOperationView` is totally different from `PointCloud2DView`. The logic in `PointCloud2DView` is not need to use in `Rect2DOperationView`